generators/ruby/faraday/templates/README.mustache

```diff
@@ -0,0 +1,104 @@
+# Onfido
+
+The official Ruby library for integrating with the Onfido API.
+
+[![Gem Version](https://badge.fury.io/rb/onfido.svg)](http://badge.fury.io/rb/onfido)
+[![Build Status](https://travis-ci.org/onfido/onfido-ruby.svg?branch=master)](https://travis-ci.org/onfido/onfido-ruby)
+
+Documentation can be found at https://documentation.onfido.com
+
+This version uses Onfido API v3.6 and is compatible with Ruby 2.4 onwards. Refer to our [API versioning guide](https://developers.onfido.com/guide/api-versioning-policy#client-libraries) for details of which client library versions use which versions of the API.
+
+## Installation & Usage
+
+### Requirements
+
+Ruby {{{generatorLanguageVersion}}}
```

Comment:
I did it as for Python but i cannot see the var `generatorLanguageVersion`  defined

Reply:
I think that's specific for python, I don't see any variable like that in [Ruby generator template for README.md](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/ruby-client/README.mustache), perhaps we can just skip the _Requirements_ section.